datasources/00_National_Data_Sets/00_04_Social_and_Economic.json

```diff
@@ -21,7 +21,6 @@
               ],
               "dataCustodian": "[Geoscience Australia](http://www.ga.gov.au/)",
               "name": "Population Estimates",
-              "legendUrl": "http://www.ga.gov.au/gis/server/capabilities/NEXIS_PopDensity_Legend.png",
```

Comment:
The generated legend is a bit misleading now, because it says "NEXIS_People_per_10000m" when really it's the number of people in each cell, the size of which changes at different zoom levels.  The old legend was probably better because at least it isn't actively misleading (it had no title).

If we wanted to be super cool, we'd generate a legend for each layer, and then show all the legends that are applicable to the current view, by comparing the scale of visible tiles to the min/max scale of each level.  But maybe just show the previous legend for now? :)